### PR TITLE
Fix tosvg in IE11 (and possibly other browsers)

### PR DESCRIPTION
**Pull request by etpinard**
@mdtusz

### CODE REVIEW

File: src/snapshot/tosvg.js [hunk 1]
```diff
@@ -107,7 +107,7 @@ module.exports = function toSVG(gd, format) {
 
     // remove draglayer for Adobe Illustrator compatibility
     if(fullLayout._draggers) {
-        fullLayout._draggers.node().remove();
+        fullLayout._draggers.remove();
```
Comment (mdtusz) [inline src/snapshot/tosvg.js]: Whoops. Didn't know that was a d3 method. Makes sense.

**Comment by mdtusz** [comment]
💃  on ✅

File: src/snapshot/tosvg.js (hunk 1 above)
Comment (etpinard) [inline src/snapshot/tosvg.js]: `ChildNode.remove` is a thing though: https://developer.mozilla.org/en-US/docs/Web/API/ChildNode/remove

but not in IE.

Comment (etpinard) [inline src/snapshot/tosvg.js]: man, we really got to get a Windows dev to wrap up https://github.com/plotly/plotly.js/issues/431 for us.

**Pull request merged**